packages/preview/peace-of-posters/0.4.0/typst.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "peace-of-posters"
+version = "0.4.0"
+authors = ["Jonas Pleyer<jonas.sci@pleyer.org>"]
+license = "MIT"
+description = "Create scientific posters in Typst."
+entrypoint = "lib.typ"
+repository = "https://github.com/jonaspleyer/peace-of-posters"
+homepage = "https://jonaspleyer.github.io/peace-of-poster/"
```

This URL is missing a `s` in `posters`.
```suggestion
homepage = "https://jonaspleyer.github.io/peace-of-posters/"
```

---

Thanks for spotting that!